test/parallel/test-stdio-pipe-access.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+require('../common');
+
+// Test if Node handles acessing process.stdin if it is a redirected
+// pipe without deadlocking
+const { spawn, spawnSync } = require('child_process');
+
+const numTries = 5;
+const who = process.argv.length <= 2 ? 'runner' : process.argv[2];
+
+switch (who) {
+  case 'runner':
+    for (let num = 0; num < numTries; ++num) {
+      console.log('Try: ' + num);
```

Comment:
if the console.log output is a part of the test, can you add a comment indicating so. If it is not, I'd prefer it to be removed.